plugins/hydra_ax_sweeper/pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[tool.towncrier]
+    package = "hydra"
+    package_dir = ""
+    filename = "NEWS.md"
+    directory = "news/"
+    title_format = "{version} ({project_date})"
+    template = "../../news/_template.rst"
+    issue_format = "[#{issue}](https://github.com/facebookresearch/hydra/issues/{issue})"
+    start_string = "<!-- TOWNCRIER -->\n"
+
+  [[tool.towncrier.type]]
+  directory = "feature"
+  name = "Features"
+  showcontent = true
+
+  [[tool.towncrier.type]]
+  directory = "api_change"
+  name = "API Change (Renames, deprecations and removals)"
+  showcontent = true
+
+  [[tool.towncrier.type]]
+  directory = "bugfix"
+  name = "Bug Fixes"
+  showcontent = true
+
+  [[tool.towncrier.type]]
+  directory = "config"
+  name = "Configuration structure changes"
+  showcontent = true
+
+  [[tool.towncrier.type]]
+  directory = "docs"
+  name = "Improved Documentation"
+  showcontent = true
+
+  [[tool.towncrier.type]]
+  directory = "trivial"
+  name = "Trivial Changes"
+  showcontent = false
+
+  [[tool.towncrier.type]]
+  directory = "maintenance"
+  name = "Maintenance Changes"
+  showcontent = true
```

Comment:
You missed my request to keep only:
feature, bugfix, api_change and config.

Do this for all plugins.

Reply:
tagged you on the previous comment